README.md

```diff
@@ -1,33 +1,93 @@
+# [Sendbird](https://sendbird.com) SyncManager for Android
 
 [![Platform](https://img.shields.io/badge/platform-android-orange.svg)](https://github.com/sendbird/sendbird-syncmanager-android)
 [![Languages](https://img.shields.io/badge/language-java-orange.svg)](https://github.com/sendbird/sendbird-syncmanager-android)
 [![Maven](https://img.shields.io/badge/maven-v1.1.28-green.svg)](https://github.com/sendbird/sendbird-syncmanager-android/tree/master/com/sendbird/sdk/sendbird-syncmanager/1.1.28)
 [![Commercial License](https://img.shields.io/badge/license-Commercial-brightgreen.svg)](https://github.com/sendbird/sendbird-syncmanager-android/blob/master/LICENSE.md)
 
-SendBird SyncManager is chat data sync management add-on for SendBird. SyncManager offers an event-based data management framework so that each view would listen data event in event handler in order to update the view. And it stores the data into SQLite which implements local caching for faster loading.
+## Table of contents
 
-## Requirements
+  1. [Introduction](#introduction)
+  1. [Before getting started](#before-getting-started)
+  1. [Getting started](#getting-started)
 
-- SendBird SyncManager works on Android 4.0+ (API level 14), Java 7+.
+<br />
 
-## Install using Gradle
+## Introduction
 
-```
+**Sendbird SyncManager** for Android is a [Chat SDK](https://github.com/sendbird/SendBird-SDK-Android) add-on that optimizes the user caching experience by interlinking the synchronization of the local data storage with the chat data in Sendbird server through an event-driven structure.
+
+### How it works
+
+SyncManager leverages local caching and synchronizes the chat data between the local storage and Sendbird server. By handling the operations in an event-driven structure, the add-on provides a simplified Chat SDK integration and a better user experience. 
+
+### Operations
+
+- **Background sync** occurs whenever there is a connection and automatically stores data fetched from Sendbird server into the local cache. 
+- **Real time sync** occurs all the time; it identifies, stores, and delivers the real-time events received from WebSocket connection. 
+- **Offline mode** ensures your client app is operational during offline mode, meaning that even without background sync, the view can display cached data. 
+
+### More about Sendbird SyncManager for Android
+
+Find out more about Sendbird SyncManager for Android on [SyncManager for Android doc](https://sendbird.com/docs/syncmanager/v1/android/getting-started/about-syncmanager). If you have any comments or questions regarding bugs and feature requests, visit [Sendbird community](https://community.sendbird.com). 
+
+<br />
+
+## Before getting started
+
+This section shows the prerequisites you need to check to use Sendbird SyncManager for Android.
+
+### Requirements 
+
+The minimum requirements for SyncManager for Android are:
+
+- `Android 4.0 (API level 14) or higher`
+- `Java 7 or higher`
+- `Gradle 3.4.0 or higher`
```

Comment:
I think it'd good to match the requirements from the docs, as below
```
Android 4.0 (API level 14) or higher
Java 8 or higher
Sendbird Chat SDK for Android 3.0.149 or higher (3.0.149 is embedded in SyncManager 1.1.27. If you want to use a higher version, you can directly declare a dependency on the higher version.)
```

Reply:
Actually, the doc should be updated as the last line should be 
"Sendbird Chat SDK for Android `3.0.150` or higher (`3.0.150` is embedded in SyncManager `1.1.28`. If you want to use a higher version, you can directly declare a dependency on the higher version.)"

Will create PR for updating the doc as well.

Reply:
Done!